frontend/src/utils/graphs/dataPreparation.ts

```
@@ -1,27 +1,32 @@
+    }
 
-  if (node.type === "directory") {
-    const newPrefix = prefix + (isLast ? "    " : "│   ");
-    node.children.forEach((child, index) => {
-      tree += createTreeStructure(child, newPrefix, index === node.children.length - 1);
-    });
-  }
+    if (child.type === "tree") {
+      const newPrefix = prefix + (isLast ? "    " : "│   ");
+      if (child.children && child.children.length) {
+        const nbChildren = child.children.length;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)